index.js

```diff
@@ -180,72 +180,92 @@
   //  Value :: Location
   var Value = 'Value';
 
-  //  _funcPath :: (Boolean, Array String, a) -> Nullable Function
-  function _funcPath(allowInheritedProps, path, _x) {
-    var x = _x;
-    for (var idx = 0; idx < path.length; idx += 1) {
-      var k = path[idx];
-      if (x == null || !(allowInheritedProps || has(k, x))) return null;
-      x = x[k];
-    }
-    return typeof x === 'function' ? x : null;
-  }
+  //  prefix :: String
+  var prefix = 'fantasy-land/';
 
   //  funcPath :: (Array String, a) -> Nullable Function
-  function funcPath(path, x) {
-    return _funcPath(true, path, x);
+  function funcPath(path, value) {
+    var x = value;
+    for (var idx = 0; idx < path.length && x != null; idx += 1) {
+      x = x[path[idx]];
+    }
+    return typeof x === 'function' ? x : null;
   }
 
   //  implPath :: Array String -> Nullable Function
   function implPath(path) {
-    return _funcPath(false, path, implementations);
+    var x = implementations;
+    for (var idx = 0; idx < path.length && has(path[idx], x); idx += 1) {
+      x = x[path[idx]];
+    }
+    return typeof x === 'function' ? x : null;
   }
 
   //  functionName :: Function -> String
-  var functionName = 'name' in function f() {} ?
+  var functionName = 'name' in Function ?
     function functionName(f) { return f.name; } :
     /* istanbul ignore next */
     function functionName(f) {
       var match = /function (\w*)/.exec(f);
       return match == null ? '' : match[1];
     };
 
+  //  getFunction :: String -> TypeRep -> Nullable Function
+  function getFunction(_name) {
+    var name = prefix + _name;
+    return function(typeRep) {
+      var f = funcPath([name], typeRep);
+      return f == null && typeof typeRep === 'function' ?
+        implPath([functionName(typeRep), name]) :
+        f;
+    };
+  }
+
+  //  getBoundMethod :: String -> a -> Nullable (a ~> b)
```

Comment:
`(a ~> b)` is cute, but inaccurate. The resulting function may be called with no arguments (since the `a` has already been bound). I think `Nullable Function` is as accurate as we can make the return type.